frontend/src/authentication/SignUp.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { signInWithGoogle } from "../firebase";
+
+const SignUp = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [displayName, setDisplayName] = useState("");
+  const [error, setError] = useState(null);
+  const createUserWithEmailAndPasswordHandler = (event, email, password) => {
+    event.preventDefault();
+    setEmail("");
+    setPassword("");
+    setDisplayName("");
+  };
```

Comment:
This currently only clears the form - I presume you'll be adding functionality here to save this information?